installer/scripts/test-rollout-status.sh

```
@@ -2,20 +2,70 @@
+    echo "
+Usage:
+    ${0##*/}
+
+Optional arguments:
+    -d, --debug
+        Activate tracing/debug mode.
+    -h, --help
+        Display this message.
+
+Example:
+    \$ export NAMESPACE="namespace"
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)